watcher/lib/influxdb/rollups.go

```diff
@@ -119,38 +119,68 @@ func buildQuery(r Rollup) (string, error) {
 	return b.String(), nil
 }
 
+// define which functions needs an additional parameter
+var funcsWithParams = []string{
+	constants.FunctionPercentile,
+	constants.FunctionBottom,
+	constants.FunctionTop,
+	constants.FunctionSample,
+}
+
+// check if function is one of the composable Functions listed above
```

Comment:
```go
// isFuncWithParams returns true if funcName names a function with parameters
```
i.e. a comment starts with the name of the function and goes on to describe its purpose